docs/features/features-walk-through/moderation.md

```diff
@@ -0,0 +1,16 @@
+# Administration panel
+
+The administration panel is a feature in InvenioRDM introduced in v10 that provides a graphical user interface for managing users and records. It is designed to be used by administrators and superusers to moderate the content of the repository.
+For more technical details you can read the [developer guide to the InvenioRDM administration panel](../../develop/topics/administration_panel.md), detailing its programmatic interface and usage.
+
+## Moderation of users and records
+
+![The User Management](img/user-management.png){ loading=lazy }
+
+_Introduced in InvenioRDM v12_
+
+The administration panel now includes a "User management" section to deactivate, block and delete users, as well as undo all those actions. Deactivating a user temporarily prevents them from creating records, but lets them appeal their deactivation. Blocked users cannot sign in anymore and deleted users have their records removed from public view and replaced with a tombstone page indicating that the user has been blocked.
```

Comment:
There is something about a hierarchy of user moderation that should be made explicit (if it's the case, and make clearly not implied if not :) ) : 

| User state | Can create records | Can sign in | Records can be seen |
| ---------- | ----------- | ------------------ | ------------------- |
| In good standing | :heavy_check_mark:  | :heavy_check_mark:  | :heavy_check_mark: |
| Deactivated | :x: | :heavy_check_mark:  | :heavy_check_mark: |
| Blocked | :x: | :x:  | :heavy_check_mark: |
| Deleted | :x: | :x:  | :x: (Tombstones) |

Right? That table could be used actually...

Reply:
Thanks for the table, I added it now with some explanations.

Reply:
On Zenodo blocked user's records are removed, but maybe this is instance specific